modin/pandas/test/test_io.py

```diff
@@ -1281,6 +1281,20 @@ def test_read_parquet(self, make_parquet_file, columns):
             columns=columns,
         )
 
+    @pytest.mark.parametrize("nrows", [40, 80, 160, 320, 640, 1280, 2560, 5120])
```

Comment:
Can we just make this `min_chunk_size * num_partitions - 1` and assert in the body of the test that `min_chunk_size < index_len < min_chunk_size * num_partitions`? That way we'll have a minimal test for this case.